src/pages/signup/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import axios from "axios";
+import Router from "next/router";
+import { Form, Input, Button } from "antd";
+import { UserOutlined, LockOutlined } from "@ant-design/icons";
+
+import style from "./signup.module.css";
+
+interface Value {
+  email: string;
+  username: string;
+  password: string;
+}
+
+const Signup = () => {
+  const auth = async (value: Value) => {
+    try {
+      await axios.post("https://reactive.loca.lt/signup/", {
+        email: value.email,
+        username: value.username,
+        password: value.password,
+      });
+      return 201;
+    } catch (err) {
+      console.error(err);
```

Comment:
log to console are not proper error handling, in such cases i usually use `status` flags, for example:

- pending - when a request has been made and we are waiting for a response
- success - when we receive the expected success response from server
- error - when we receive an error as a response, usually anything with status code 400 +

an depending on the status, we can render different things